lighthouse-core/test/gather/gatherers/seo/font-size-test.js

```diff
@@ -13,47 +13,83 @@ let fontSizeGather;
 const smallText = ' body small text ';
 const bigText = 'body big text';
 const failingText = 'failing text';
-const bodyNode = {nodeId: 3, nodeName: 'BODY', parentId: 1};
-const failingNode = {nodeId: 10, nodeName: 'P', parentId: 3};
+const bodyNode = {nodeId: 3, backendNodeId: 102, nodeName: 'BODY', parentId: 1, fontSize: '10px'};
+const failingNode = {nodeId: 10, backendNodeId: 109, nodeName: 'P', parentId: 3};
 const nodes = [
-  {nodeId: 1, nodeName: 'HTML'},
-  {nodeId: 2, nodeName: 'HEAD', parentId: 1},
+  {nodeId: 1, backendNodeId: 100, nodeName: 'HTML'},
+  {nodeId: 2, backendNodeId: 101, nodeName: 'HEAD', parentId: 1},
   bodyNode,
   {
     nodeId: 4,
+    backendNodeId: 103,
     nodeValue: 'head text',
     nodeType: FontSizeGather.TEXT_NODE_TYPE,
     parentId: 2,
   },
   {
     nodeId: 5,
+    backendNodeId: 104,
     nodeValue: smallText,
     nodeType: FontSizeGather.TEXT_NODE_TYPE,
     parentId: 3,
   },
-  {nodeId: 6, nodeName: 'H1', parentId: 3},
+  {nodeId: 6, backendNodeId: 105, nodeName: 'H1', parentId: 3},
   {
     nodeId: 7,
+    backendNodeId: 106,
     nodeValue: bigText,
     nodeType: FontSizeGather.TEXT_NODE_TYPE,
     parentId: 6,
   },
-  {nodeId: 8, nodeName: 'SCRIPT', parentId: 3},
+  {nodeId: 8, backendNodeId: 107, nodeName: 'SCRIPT', parentId: 3},
   {
     nodeId: 9,
+    backendNodeId: 108,
     nodeValue: 'script text',
     nodeType: FontSizeGather.TEXT_NODE_TYPE,
     parentId: 8,
   },
   failingNode,
   {
     nodeId: 11,
+    backendNodeId: 110,
     nodeValue: failingText,
     nodeType: FontSizeGather.TEXT_NODE_TYPE,
     parentId: 10,
   },
 ];
 
+const stringsMap = {};
+const strings = [];
+const dedupe = (value) => {
```

Comment:
WDYT about something like `getOrCreateStringRef`/getOrCreateStringIndex` to indicate the main reason we're doing this is so that we're dealing with that `.strings` property